pkg/contracts/src/RegistryCommunity.sol

```diff
@@ -154,25 +155,31 @@ contract RegistryCommunity is ReentrancyGuard, AccessControl {
         address _strategyTemplate;
     }
 
-    //TODO: can change to uint32 with optimized storage order
-    FAllo public allo;
-    IRegistry public registry;
-    IERC20 public gardenToken;
+    //DONE: grouping and order of variables
+    // Smaller uints and bools packed together
+    bool public isKickEnabled; // 1 byte, consider aligning with other small data types or bools if available
```

Comment:
bool use 32 bytes not 1

Reply:
Noo bool is 1 byte ( 8 bits )

Reply:
i express bad, what i mean its for efficient its worst than use 32 bytes.

check out

![image](https://github.com/1Hive/gardens-v2/assets/691510/bb39b07e-5975-4f4b-bdc5-e0681406a97b)
https://docs.soliditylang.org/en/v0.8.17/internals/layout_in_storage.html

But lets keep like that for now, in upgradable stuff we can improve it later